_posts/0000-01-02-JeremDev21.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title:"welcome to our second slide!"
+---
+I'm right now learning how to use GitHub,I'm here to learn coding
```

Comment:
I'm right now learning how to use GitHub,I'm here to learn coding,and discover many programmation languages.